relay-server/src/actors/outcome.rs

```diff
@@ -707,7 +707,7 @@ impl KafkaOutcomesProducer {
     /// Creates and connects the Kafka producers.
     ///
     /// If the given Kafka configuration parameters are invalid, or an error happens during
-    /// connecting during the broker, an error is returned.
+    /// connecting the broker, an error is returned.
```

Comment:
Let's make it "while connecting the broker" then:

> We use during before nouns and noun phrases but not before verbs

https://dictionary.cambridge.org/grammar/british-grammar/during